experimental/framework/frs-client/src/test/CreateClient.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { generateUser } from "@fluidframework/server-services-client";
+import {
+    FrsClient,
+    FrsConnectionConfig,
+    InsecureTokenProvider,
+} from "..";
+
+export class CreateClient {
+    // use FrsClient remote mode will run against live Frs service. Default to running Tinylicious for PR validation
+    // and local testing so it's not hindered by service availability
+    private readonly useFrs: boolean;
+    private readonly user: any;
+    private readonly tenantKey: string;
+    private readonly connectionConfig: FrsConnectionConfig;
+
+    constructor() {
+        this.useFrs = process.env.FLUID_CLIENT === "frs";
+        this.tenantKey = this.useFrs ? process.env.fluid__webpack__tenantKey as string : "";
```

Comment:
How is the `process.env.fluid__webpack__tenantKey` being set?

Reply:
It's being set with `"test:frs": "cross-env process.env.FLUID_CLIENT='\"frs\"' && npm run test:realsvc:run"`